narratives/ncov_sit-rep_2020-04-03.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Genomic analysis of COVID-19 spread. Situation report 2020-04-03.
+authors:
+  - Emma Hodcroft
+  - Cassia Wagner
+  - Sidney M. Bell
+  - Nicola Müller
+  - James Hadfield
+  - Louise Moncla  
+  - Richard Neher
+  - Trevor Bedford
+authorLinks:
+  - https://neherlab.org/emma-hodcroft.html
+  - https://bedford.io/team/cassia-wagner/
+  - https://twitter.com/sidneymbell
+  - https://bedford.io/team/nicola-mueller/
+  - https://bedford.io/team/james-hadfield/
+  - https://bedford.io/team/louise-moncla/
+  - https://neherlab.org/richard-neher.html
+  - https://bedford.io/team/trevor-bedford/
+affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; CZI, CA, USA"
+translators:
+translatorLinks:
+date: "2020 April 03"
+dataset: "https://nextstrain.org/ncov/2020-04-02?d=map&legend=closed"
+abstract: "This report uses publicly shared genomic data to track the spread of COVID-19. These reports are updated weekly."
+---
+<!-- Translators: Only text after : in the above ^ needs to be translated -->
+<!-- Comment tags like these do not need to be translated, they are only to help you! -->
+<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
+<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->
+
+<!-- This is left-side text 1-->
+# [Table of Contents](https://nextstrain.org/ncov/2020-04-02?d=tree,map&p=grid)
+
+* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=2).     
+* [A note on sampling](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=3).
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=4).
+<!-- * [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=7). -->
+* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=8).
+* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=11).
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=13).
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=14).
+
+<!-- This is right-side text -->
+```auspiceMainDisplayMarkdown
+# Executive summary
+
+We analyzed 2,544 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading within communities. In this week's updates, we report:
```

Comment:
```suggestion
We analyzed 2,544 publicly shared SARS-CoV-2/hCoV-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading within communities. In this week's updates, we report:
```
I've had translators pointing this out in the last report. I think it's worth being precise here. I'm still not sure what the lab's decided name is for the virus though.